src/integration.test.ts

```diff
@@ -297,6 +298,43 @@ describe(`Integration test for portal '${portal}'`, () => {
       expect(returnedEntry.data).toEqualUint8Array(dataLinkBytes);
     });
 
+    it("should set and get entry data", async () => {
+      const { publicKey, privateKey } = genKeyPairAndSeed();
+      const data = new Uint8Array([1, 2, 3]);
+
+      // Set the entry data.
+      await client.db.setEntryData(privateKey, dataKey, data);
+
+      // Get the entry data.
+      const { data: returnedData } = await client.db.getEntryData(publicKey, dataKey);
+
+      // Assert the returned data equals the original data.
+      expect(returnedData).toEqualUint8Array(data);
+    });
+
+    it("should set and delete entry data", async () => {
+      const { publicKey, privateKey } = genKeyPairAndSeed();
+      const data = new Uint8Array([1, 2, 3]);
+
+      // Set the entry data.
+      await client.db.setEntryData(privateKey, dataKey, data);
+
+      // Delete the entry data.
+      await client.db.deleteEntryData(privateKey, dataKey);
+
+      // Trying to get the deleted data should result in null.
+      const { data: returnedData } = await client.db.getEntryData(publicKey, dataKey);
+      // TODO: Should this equal null?
```

Comment:
I think it should equal whatever is returned if the entry were to be not set in the first place, so I guess that's `null` idd.

Reply:
Hm, `deleteEntryData` is basically setting the record to a new `Uint8Array`:
```js
await this.setEntryData(path, new Uint8Array(RAW_SKYLINK_SIZE), customOptions);
```
So, I don't think it would be `null`, it would rather be a valid array with all zeros, unless there's some magic happening somewhere which I have missed.

Reply:
I personally think it should probably act like getJSON in that regard, so that deleteEntryData actually appears to have deleted the entry. I'll document this as per your other comment.